src/editor/editorinteractive.cc

```diff
@@ -416,26 +421,33 @@ void EditorInteractive::rebuild_showhide_menu() {
 	                  g_image_cache->get("images/wui/menus/toggle_buildhelp.png"), false, "",
 	                  shortcut_string_for(KeyboardShortcut::kCommonBuildhelp));
 
+	/** TRANSLATORS: An entry in the editor's show/hide menu to toggle whether maximum building spaces are
+	 * shown */
+	showhidemenu_.add(get_display_flag(dfShowMaximumBuildhelp) ? _("Hide Maximum Building Spaces") : _("Show Maximum Building Spaces"),
+	                  ShowHideEntry::kMaximumBuildingSpaces,
+	                  g_image_cache->get("images/wui/menus/toggle_buildhelp.png"), false, "",
```

Comment:
```suggestion
	                  g_image_cache->get("images/wui/menus/toggle_buildhelp.png"), false,
	                  _("Toggle whether to show maximum building spaces that will be available if all immovables (trees, rocks, etc.) are removed"),
```
(tooltip text -- I hope I didn't mess it up)